app/views/admin/missing_copyright.html.haml

```diff
@@ -6,13 +6,11 @@
     %tr
       %th= t(:name)
       %th= t(:life_years)
-      %th= t(:deathdate)
```

Comment:
I've also removed separate deathdate column, as deathdate is displayed in life_years